gpg_key.go

```diff
@@ -15,6 +15,9 @@ var _ GPGKeys = (*gpgKeys)(nil)
 //
 // TFE API Docs: https://www.terraform.io/cloud-docs/api-docs/private-registry/gpg-keys
 type GPGKeys interface {
+	// Lists GPG keys in a private registry.
+	List(ctx context.Context, registryName RegistryName, options *GPGKeyListOptions) (*GPGKeyList, error)
```

Comment:
If filter[namespace] is required, we should not make this a pointer in order to reinforce that, I think

Reply:
I was iffy on this and decided on consistency (our ListOpts are almost always pointers). But yeah, I can imagine this confusing someone so I'll change.